### DISCUSSION TITLE
v5.0

### DISCUSSION THREAD

**Discussion by pjaaskel** [pocl/pocl#1375] [Announcements]
Release notes are now available [within the docs](http://portablecl.org/docs/html/notes_5_0.html).

<hr /><em>This discussion was created from the release <a href='https://github.com/pocl/pocl/releases/tag/'>v5.0</a>.</em>

**Comment by trholding** [comment]
The Doc mentions:

> The PoCL-Remote client has successfully been run on PC and Android devices and with some tweaks (dubbed “Nano-PoCL”) it has proven usable even in bare metal firmware of low end embedded devices with limited toolchain support. 

Where can I get more info about Nano-PoCL?

**Reply by pjaaskel** [reply]
That work is being polished (PoCL-R client running on Crazyflie/AI-deck/RTOS) and we will upstream it gradually. There's a master's [thesis](https://trepo.tuni.fi/handle/10024/143028) of this work and an IWOCL [poster](https://dl.acm.org/doi/10.1145/3585341.3585376) and an EU project [presentation](https://www.youtube.com/watch?v=GtNsznRk5jE). Using PoCL-R from Android support is now [documented](https://github.com/pocl/pocl/blob/main/doc/sphinx/source/android.rst), it can be used also with JOCL - the Java OpenCL bindings.

**Reply by trholding** [reply]
Thank you, its impressive, can't wait for the up streaming.
